README.md

```diff
@@ -1,17 +1,119 @@
+However, it is very much in a usable state, so if you're fine living on the bleeding edge, give it a try. 
 
-Testers2 will be split into this repository during development to avoid the need for users to run a custom chisel3 branch.
-Independent JARs will also be published, so users can use managed dependencies.
 
-However, the code will be merged back into chisel3 once there is a final 'release', as chisel3 needs a comprehensive test framework.
+## Overview
+Testers2 is a test harness for [Chisel](freechipsproject/chisel3)-based RTL designs, currently supporting directed testing (all test stimulus manually specified - no constrained random and coverage-driven flows).
+Testers2 emphasizes tests that are lightweight (minimizes boilerplate code), easy to read and write (understandability), and compose (for better test code reuse).
+
+The core primitives are similar to nonsynthesizable Verilog: input pin assignment (`poke`), pin value assertion (`expect`), and time advance (`step`). Threading concurrency is also supported with the use of `fork` and `join`, and concurrent accesses to wires are checked to prevent race conditions.
+
+### Migrating from chisel-testers
+The core abstractions (`poke`, `expect`, `step`) are similar to [chisel-testers](freechipsproject/chisel-testers), but the syntax is inverted: instead of doing `tester.poke(wire, value)` with a Scala number value, in testers2 you would write `write.poke(value)` with a Chisel literal value.
+Furthermore, as no reference to the tester context is needed, test helper functions can be defined outside a test class and written as libraries.
+
+Currently, this should support all the functionality that was in chisel-testers, and provides additional features.
+This project is meant to supersede chisel-testers, and eventually may become a default part of chisel3.
+
+Test cases written in chisel-testers cannot be directly used in testers2, as the syntax is significantly different.
+
+
+## Getting Started
+
+### Installation 
+To use chisel-testers as a managed dependency, add this in your build.sbt:
+```scala
+libraryDependencies += "edu.berkeley.cs" %% "chisel-testers2" % "0.1-SNAPSHOT"
+```
+
+Note, chisel-testers2 snapshots generally track chisel3 snapshots, and requires the use of a chisel3 snapshot.
+We may introduce versioned snapshots and releases in the future, tied to a particular release of chisel3.
+
+You can also build testers2 locally with `publishLocal`.
+
+### Writing a Test
+Testers2 integrates with the [ScalaTest](scalatest.org) framework, which provides a framework for detection and execution of unit tests.
+
+Assuming a typical Chisel project, create a new file in `src/test/scala/`, for example, `BasicTest.scala`.
+
+In this file:
+1.  Add the necessary imports:
+    ```scala
+    import org.scalatest._
+    import chisel3._
+    import chisel3.tester._
```

Comment:
I think you should have here (or separately) `import chisel3.experimental.BundleLiterals._`
similar arguments could be made for `import chisel3.tester.experimental.TestOptionBuilder._`
Both are reasonably likely to be necessary and finding them is not obvious, including IntelliJ's otherwise
robust auto-importing tools

Reply:
added experimental imports in the bundle literals section, and a Verilator section